daemon/main.go

```diff
@@ -111,6 +112,7 @@ func main() {
 	//r.DELETE("/v1/p/:id", access.IfAuth(deleteSnippet))
 	r.PUT("/v1/p", access.IfAuth(client, access.Control(updateSnippet, access.Update)))
 	r.POST("/v1/worked", access.IfAuth(client, snippetWorked))
+	r.POST("/v1/slack", slackCommand)
```

Comment:
No, because even if we support more command this same endpoint will take care of it. The payload sent by slack allow us to do that.